docs/team/sltsheryl.md

```diff
@@ -4,37 +4,63 @@ title: Sheryl-Lynn Tan's Project Portfolio Page
 ---
 
 ### Project: Mass Linkers
-
-Mass Linkers is a Desktop application tool for CS2103T students in a tutorial group. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about [to be added soon] kLoC.
+Mass Linkers is a powerful Desktop application tool that provides a centralised platform for Computer Science (CS) students to find study support from batchmates with common modules. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
 
 Given below are my contributions to the project.
 
-* **New Feature**: [To be added soon]
-    * What it does: [To be added soon]
-    * Justification: [To be added soon]
-    * Highlights: [To be added soon]
-    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+####New Enhancement: User Interface
+
```

Comment:
Not sure if we need to follow the format given, i.e. What it does, Justification, Highlights, Credits.
Refer to https://se-education.org/addressbook-level3/team/johndoe.html for examples.

Reply:
Edited. 
Kept the New Feature as a header and not a bullet, cuz it's neater this way